docs/admin-guide/README.md

```diff
@@ -18,6 +18,7 @@ This includes instructions for bioinformaticians to install, manage and update a
   * [Chanjo coverage integration](./chanjo_coverage_integration.md)
   * [Loqusdb integration](./loqusdb_integration.md)
   * [Matchmaker Exchange integration](./matchmaker_exchange_integration.md)
+  * [BioNano Genomics Access FSHD integration](./bionano_integration.md)
```

Comment:
The cod page is missing but can be fixed later! 👍🏻

Reply:
Ahem, it existed locally. It its not email attachments..

Reply:
For cod (🐟 ) I mean doc you know..